### DISCUSSION TITLE
Define an up-limit space for each pool will bring fairness for common Chia miners

### DISCUSSION THREAD

**Discussion by JamesDJchia** [Chia-Network/chia-blockchain#3449] [General]
The top farmer earn 3,089 (30.89%) (502.42 PiB) is overwhelmed smaller miners/farmers around the world.

https://www.chiaexplorer.com/top-farmers

It is reasonable to Define an up-limit space for each pool.  Say, less than 10%.

This will bring fairness for common Chia miners, and prevent 51% attack risk for Chia robustness.

**Comment by quixot1c** [comment]
(Disclaimer: not affiliated with Chia) I think this is not going to help, as this party that now has that much drive space plotted could just make multiple private keys and pretend to be multiple parties, while still controlling the same portion of the network.

**Reply by chiabug** [reply]
I agree.
